hs_app_netCDF/receivers.py

```diff
@@ -227,21 +232,80 @@ def netcdf_pre_add_files_to_resource(sender, **kwargs):
                     f.resource_file.delete()
                     f.delete()
 
-            # TODO: update resource modification info
-            # user = kwargs['user']
-            # resource_modified(resource, user)
+            # update resource modification info
+            user = kwargs['user']
+            utils.resource_modified(nc_res, user)
 
             # extract metadata
             try:
                 res_dublin_core_meta = nc_meta.get_dublin_core_meta(nc_dataset)
-            except:
+            except Exception:
                 res_dublin_core_meta = {}
 
             try:
                 res_type_specific_meta = nc_meta.get_type_specific_meta(nc_dataset)
-            except:
+            except Exception:
                 res_type_specific_meta = {}
 
+            # update title info
```

Comment:
If this PR is to add tests, please explain the additional functionality represented here.